_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why it is so important to do analysis pre-GSoC and
```

Comment:
I am not sure of this sentence.
Shouldn't it be `pre-GSoC analysis`, instead of `analysis pre-GSoC`?
Not quite sure about it.

Reply:
I think the question should be rephrased to something like this `What is a cEP and why it is so important to do pre-GSoC analysis and design it as early as possible, to be finalized as a cEP?`

Reply:
Also `why it is` can also be written as `why is it`
will it look better ?

Reply:
My opinion:
`What is a cEP and why is it important to do a pre-GSoC analysis ...`
What's your thought?

Reply:
My thought == Your opinion.
Will make changes ASAP by evening 😇

Reply:
Pushed to new changes 👍